packages/adapter-drizzle/src/index.ts

```diff
@@ -28,23 +28,23 @@ import { DefaultSchema, SqlFlavorOptions } from "./lib/utils.js"
 import type { Adapter } from "@auth/core/adapters"
 
 export {
-  postgresUsersTable,
+  mysqlAccountsTable,
+  mysqlSessionsTable,
+  mysqlUsersTable,
+  mysqlVerificationTokensTable,
```

Comment:
why are these exported 🤔

Reply:
If users want to use default tables they can just import them instead of copying

Reply:
let's not, for now. it wasn't there before. let's minimize the public api surface